sdk/examples/how_tos/native_tokens/mint.rs

```diff
@@ -33,13 +33,14 @@ async fn main() -> Result<()> {
         .finish()
         .await?;
     let account = wallet.get_account("Alice").await?;
+    account.sync(None).await?;
 
     // Set the stronghold password
     wallet
         .set_stronghold_password(var("STRONGHOLD_PASSWORD").unwrap())
         .await?;
 
-    println!("Sending alias output transaction...");
+    println!("Preparing alias output transaction...");
 
     // First create an alias output, this needs to be done only once, because an alias can have many foundry outputs
```

Comment:
This comment is not in the other language examples, should we also maybe change it to check in the balance if we have an alias output and only create a new one if not?

Reply:
No we could do that 🤔

Reply:
LGTM